arbnode/sequencer.go

```diff
@@ -413,6 +413,13 @@ func (s *Sequencer) createBlock(ctx context.Context) (returnValue bool) {
 		return false
 	}
 	if err != nil {
+		if errors.Is(err, context.Canceled) {
+			// thread closed. We'll later try to forward these messages.
+			for _, item := range queueItems {
+				s.requeueOrFail(item, err)
+			}
+			return false
```

Comment:
I'd return true here to avoid hot looping waiting for this error to stop being returned